packages/localize/examples/transform/src/locale-codes.ts

```diff
@@ -1,9 +1,3 @@
-/**
```

Comment:
Accidental deletion?

Reply:
Actually intentional -- this license header got caught by the script I wrote the other week when we updated all of our license headers in the whole monorepo to the new style. However, this file is a generated artifact of lit localize, it doesn't get generated with a copyright header (maybe that needs to be an option or something for the very license header pedantic but... whatever).